### PR TITLE
Fixed a bug where saving child items of a storm...

### PR DESCRIPTION
**Pull request by zmoore-usgs**
... aggregation would cause an error to be thrown due to an invalid attempt to delete the NHC Track item.

### CODE REVIEW

File: coastal-hazards-liquibase/pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
 		<pluginRepository>
 			<id>cida-public</id>
 			<name>CIDA plugin repository</name>
-			<url>http://cida.usgs.gov/maven/cida-public/</url>
+			<url>https://cida.usgs.gov/maven/cida-public/</url>
```
Comment (isuftin) [inline coastal-hazards-liquibase/pom.xml]: Don't want to switch everything else to https while you're in here?

Comment (isuftin) [reply coastal-hazards-liquibase/pom.xml]: Oops. Didn't scroll

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/ItemResource.java [hunk 2]
```diff
@@ -181,56 +182,54 @@ public Response postItem(String content, @Context HttpServletRequest request) {
...
 			}
-			try (StatusManager statusMan = new StatusManager(); ThumbnailManager thumbMan = new ThumbnailManager()) {
-				Status status = new Status();
-				status.setStatusName(Status.StatusName.ITEM_UPDATE);
-				statusMan.save(status);
+		}
 
-				thumbMan.updateDirtyBits(id);
-			}
+		if (null != mergedId) {
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/ItemResource.java]: Would it make sense to move this above line 213? After all, the if block there won't run if mergedId is null and you can throw early out of that larger try block

Comment (isuftin) [reply coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/ItemResource.java]: Also, if you do that, on the if block on line 214 you then don't have to double-check for mergedId being null.

Comment (zmoore-usgs) [reply coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/ItemResource.java]: Yeah, that makes sense. It seemed kind of funky to explicitly throw something while within a try block, but that's really not an issue.

Comment (isuftin) [reply coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/ItemResource.java]: Yeah the JDK made things a little confusing when combining the functionality of try/catch with try-with-resources :/

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/ItemResource.java [hunk 3]
```diff
@@ -210,23 +210,22 @@ public Response updateItem(@Context HttpServletRequest request, @PathParam("id")
 				mergedId = itemManager.merge(mergedItem);
 			}
 
+			if (null != mergedId) {
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/ItemResource.java]: 👍

**Comment by mhines-usgs** [comment]
Pulling this locally to check out!

**Comment by mhines-usgs** [comment]
works great for me locally, let's merge and test on dev.

**Pull request merged**